tensorflow/python/keras/testing_utils.py

```diff
@@ -93,7 +93,9 @@ def layer_test(layer_cls, kwargs=None, input_shape=None, input_dtype=None,
       string or integer values.
     adapt_data: Optional data for an 'adapt' call. If None, adapt() will not
       be tested for this layer. This is only relevant for PreprocessingLayers.
-
+    custom_objects: Optional Objects for your custom layer. If you write a 
+      your layer, you can use this variable.
```

Comment:
nit: custom_objects: Optional dictionary mapping name strings to custom objects in the layer class. This is helpful to test a custom layer.

Reply:
Thanks your help!

Reply:
I think @yhliang2018 meant that you should update this description to instead say:

```
custom_objects: Optional dictionary mapping name strings to custom objects in the layer class. This is helpful for testing custom layers.
```

Then we can go ahead and approve this.